src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,6 +121,9 @@ public static DateTime parseDateTime(String dateTime) throws ParseException {
         requireNonNull(dateTime);
         String trimmedDateTime = dateTime.trim();
         if (!DateTime.isValidDateTime(trimmedDateTime)) {
+            if (DateTime.isValidFormat(trimmedDateTime)) {
+                throw new ParseException(DateTime.MESSAGE_INVALID_DATETIME);
+            }
```

Comment:
not very sure what the problem is but it seems like isValidFormat is not catching all invalid formats. i tested s/22 oct 2022 and a DateTimeParseException is thrown at 129.

Reply:
22 oct 2022 is a valid format, that's why `isValidFormat` is not catching it. This issues arises only because `DateTimeFormatter(d/MMM/yyyy)` requires the first letter of the month to be in uppercase, and the other letters to be lower case. I have fixed this.